bot/exts/valentines/valentine_zodiac.py

```diff
@@ -1,7 +1,10 @@
+import calendar
 import logging
 import random
+from datetime import datetime
```

Comment:
use date instead of DateTime as we have nothing to do with DateTime.

Reply:
Actually I am taking date as param in command so if I use date directly it messes up the code

Reply:
`import datetime` -> `datetime.date` or `from datetime import date as datetime` etc.